tests/lib/config/config-initializer.js

```diff
@@ -66,16 +66,12 @@ describe("configInitializer", function() {
         sh.mkdir("-p", fixtureDir);
         sh.cp("-r", "./tests/fixtures/config-initializer/.", fixtureDir);
         fixtureDir = fs.realpathSync(fixtureDir);
+        cwd = process.cwd();
+        process.chdir(fixtureDir);
```

Comment:
I think this could be accomplished in `npm-util` tests by stubbing `fs.ReadFileSync`. Let me try that.